packages/edit-navigation/src/store/transform.js

```diff
@@ -84,41 +212,20 @@ function mapMenuItemsToBlocks( menuItems ) {
 	};
 }
 
-/**
- * A WP nav_menu_item object.
- * For more documentation on the individual fields present on a menu item please see:
- * https://core.trac.wordpress.org/browser/tags/5.7.1/src/wp-includes/nav-menu.php#L789
- *
- * Changes made here should also be mirrored in packages/edit-navigation/src/store/utils.js.
```

Comment:
Just noting this. Much of this code is mirrored in the block so we should consider that.

Reply:
Block library seems to only use it to create a placeholder. Perhaps we could remove it from there? After all, blocks are generic and independent of WordPress.

Reply:
Tracked in https://github.com/WordPress/gutenberg/issues/35037